lerobot/scripts/control_robot.py

```diff
@@ -115,6 +115,14 @@
 # Utilities
 ########################################################################################
 
+def say(text):
+    #check if mac, linux, or windows.
+    if platform.system() == "Darwin":
+        os.system(f'say "{text}" &')
+    elif platform.system() == "Linux":
+        os.system(f'spd-say "{text}" &')
+    elif platform.system() == "Windows":
+        os.system(f'PowerShell -Command "Add-Type -AssemblyName System.Speech; (New-Object System.Speech.Synthesis.SpeechSynthesizer).Speak(\'{text}\')" &')
```

Comment:
Could you add `blocking`?
```suggestion
def say(text, blocking=False):
    # Use the ampersand to run command in the background
    suffix = "" if blocking else " &"

    # Check if mac, linux, or windows.
    if platform.system() == "Darwin":
        os.system(f'say "{text}"{suffix}')
    elif platform.system() == "Linux":
        os.system(f'spd-say "{text}"{suffix}')
    elif platform.system() == "Windows":
        os.system(f'PowerShell -Command "Add-Type -AssemblyName System.Speech; (New-Object System.Speech.Synthesis.SpeechSynthesizer).Speak(\'{text}\')"{suffix}')
```